test/integration/accountSwitch.test.js

```diff
@@ -102,7 +101,7 @@ describe('@integration: Account switch', () => {
       localStorageStub.withArgs('lastActiveAccountIndex').returns(1);
     });
 
-    step('Given I\'m on "account switcher" with accounts: "genesis,delegate,empty account"', setupStep);
+    step('Given I\'m on "account switcher" with accounts: "genesis,delegate,second passphrase account"', setupStep);
```

Comment:
In the code above you're replacing `delegate` with `second passphrase account`, so here it should be `genesis,second passphrase account,empty account`